test/fixtures/output/iam/ip-group/update.golden

```diff
@@ -0,0 +1,5 @@
++-------------+------------------------------+
+| ID          | ipg-wjnde                    |
+| Name        | new-demo-group               |
+| CIDR blocks | 1.2.3.4/12, 147.150.200.0/24 |
```

Comment:
I'm looking for other examples of this and I can't find `.golden` files related to the CLI logger. In my tests, the CLI warnings don't go to the terminal, and therefore don't get compared against the `.golden`. Is there a way to change this? Or is there a fundamentally different way to test these?

Reply:
Discussed offline

Reply:
Note: updated this to use `output.ErrPrintf` rather than passing in the CLI logger. Pushing new changes that include tests for these 3 errors on the IP filters and groups `update` command.